### PR TITLE
feat(data): add static data, update workflow templates and config

### PR DESCRIPTION
**Pull request by andreafinlay**

### CODE REVIEW

File: src/data/planets.json [hunk 1]
```diff
@@ -0,0 +1,259 @@
+[
```
Comment (MichaelMansourati) [inline src/data/planets.json:1]: this will probably work better as an object keyed by planet name

**Review by MichaelMansourati — COMMENTED** [review]
probably any static data structures should be objects with names for keys for each entity i.e planet name is key for data for that planet

**Review by MichaelMansourati — CHANGES_REQUESTED** [review]
re-orienting review as Change Request

**Comment by andreafinlay** [comment]
@MichaelMansourati data updated!

**Pull request merged**